### PR TITLE
CLDR-14253 Spec improvements: compound unit pattern algorithms

### PR DESCRIPTION
**Pull request by macchiati**
These are based on https://github.com/unicode-org/cldr/pull/1142

##### Checklist

- [ ] Issue filed: https://unicode-org.atlassian.net/browse/CLDR-14253
- [ ] Updated PR title and link in previous line to include Issue number

### CODE REVIEW

File: docs/ldml/tr35-general.md [hunk 1]
```diff
@@ -1204,15 +1204,15 @@ If there is no precomputed form, the following process in pseudocode is used to
...
 1. return the element value in the locale for the path corresponding to the key, locale, length, and variants — using normal inheritance including [Lateral Inheritance](https://unicode-org.github.io/cldr/ldml/tr35.md#Multiple_Inheritance) and [Parent Locales](https://unicode-org.github.io/cldr/ldml/tr35.md#Parent_Locales).
 
-**Extract(corePattern, coreUnit, placeHolder, placeholderPosition)**
+**Extract(corePattern, coreUnit, placeholder, placeholderPosition)**
 
-1. Find the position of the **placeHolder** in the core pattern
-2. Set **placeHolderPosition** to that postion (start, middle, or end)
-3. Remove the **placeHolder** from the **corePattern** and set **coreUnit** to that result
+1. Find the position of the **placeholder** in the core pattern
+2. Set **placeholderPosition** to that postion (start, middle, or end)
```
Comment (hugovdm) [inline docs/ldml/tr35-general.md:1210]:
```suggestion
2. Set **placeholderPosition** to that position (start, middle, or end)
```

**Review by hugovdm — APPROVED** [review]
I suppose I'll have to re-approve after the position typo fix?

**Comment by macchiati** [comment]
Yes, I just fixed position. Can you approve again?

File: docs/ldml/tr35-general.md (hunk 1 above)
Comment (macchiati) [reply docs/ldml/tr35-general.md]: Just fixed, can you rereview?

**Review by hugovdm — APPROVED** [review]

**Review by hugovdm — APPROVED** [review]

**Review by pedberg-icu — APPROVED** [review]

**Pull request merged**